packages/amplify-codegen/src/commands/statements.js

```diff
@@ -30,25 +39,22 @@ async function generateStatements(context, forceDownloadSchema, maxDepth, withou
       return;
     }
   }
-  if (!projects.length && withoutInit) {
-    context.print.info(constants.ERROR_CODEGEN_NO_API_CONFIGURED);
-    return;
-  }
 
   for (const cfg of projects) {
     const includeFiles = path.join(projectPath, cfg.includes[0]);
     const opsGenDirectory = cfg.amplifyExtension.docsFilePath
       ? path.join(projectPath, cfg.amplifyExtension.docsFilePath)
       : path.dirname(path.dirname(includeFiles));
     const schemaPath = path.join(projectPath, cfg.schema);
-    let region;
     let frontend;
```

Comment:
Is this reassigned at all, or can this be converted into a ternary `const` assignment?

```ts
const frontend = withoutInit ? cfg.amplifyExtension.frontend : getFrontEndHandler(context);
```